package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I don't think you should be commiting any new files on the root level. This `package-lock.json` appears like an error.